packages/contracts/src/plugins/token/IMerkleMinter.sol

```diff
@@ -29,17 +29,6 @@ interface IMerkleMinter {
     /// @notice The address of the `MerkleDistributor` to clone from.
     function distributorBase() external returns (IMerkleDistributor);
 
-    /// @notice Initializes the MerkleMinter.
-    /// @dev This method is required to support [ERC-1822](https://eips.ethereum.org/EIPS/eip-1822).
-    /// @param _dao The IDAO interface of the associated DAO.
-    /// @param _token A mintable [ERC-20](https://eips.ethereum.org/EIPS/eip-20) token.
-    /// @param _distributorBase A `MerkleDistributor` to be cloned.
-    function initialize(
```

Comment:
Is there any disablerInitialize there that should be done?

Reply:
It is already in the constructor of the implementation code in `MerkleMinter`.